### ISSUE TITLE
service.d avc denied for mount

### ISSUE THREAD

**Issue by aIecxs** [topjohnwu/Magisk#5476]
Device: `Poco M3`
Android version: `10`
Magisk version name: `1a1b346c`
Magisk version code: `24102`

mount script from `service.d` is working but I get avc denied for `mount`

```
[   29.878791] type=1400 audit(1645281544.882:1577): avc: denied { ioctl } for comm="busybox" path="/mnt/media_rw/4D47-9860/.data.sdext2.img" dev="mmcblk0p1" ino=1862 ioctlcmd=0x1272 scontext=u:r:magisk:s0 tcontext=u:object_r:vfat:s0 tclass=file permissive=1
```
It does not happen when prefix /system/bin/mount is used. I could not track it down to specific version but same clean install (no modules)

[magisk_log_2022-02-19T15:39:22.log](https://github.com/topjohnwu/Magisk/files/8102452/magisk_log_2022-02-19T15.39.22.log)
[dmesg.log](https://github.com/topjohnwu/Magisk/files/8102457/dmesg.log)

mount script
[https://chat.stackoverflow.com/transcript/241847](https://chat.stackoverflow.com/transcript/241847/2022/2/10)

**Comment by aIecxs** [comment]
yes because added setenforce 0 doubt that's intended

**Comment by aIecxs** [comment]
@yujincheng08 please assist on stackoverflow chat. how can we make mount script work? no problem on android 9 but on android 10 dmesg is full of avc denied untrusted app - this doesn't look normal for stock kernel

**Comment by aIecxs** [comment]
btw file names must not contain `:` colons `adb pull /sdcard` on Windows always fails because of magisk_log_2022-02-19T15:39:22.log that's really annoying

**Comment by aIecxs** [comment]
once there was a time where such issues wasn't closed that fast 4dacffd

**Comment by aIecxs** [comment]
@linuxandria do you know where to place `supolicy --live` script can you give example for custom module
